### PR TITLE
typos found while reviewing for style

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** typos found while reviewing for style variations 
- **Is this PR related to a Monolith release?** No

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/concepts/okta-hosted-flows/index.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Okta hosts authentication and identity flows for your applications. This means t
 
 You can build and self-host these flows in your application if you'd like, but using Okta-hosted flows has some advantages:
 
-* **Works out of the box:** The sign-in page Okta provides works by default, not requiring any configuration on your part. If you do want to change it, we provide a UI editor with preview, as well as a number of macros to help you customize the page to your liking. For more information see the [Customize the Hosted Sign-in Page](/docs/guides/custom-hosted-signin/).
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/docs/concepts/okta-hosted-flows/index.md]: For more information**, see the** ...

File: packages/@okta/vuepress-site/docs/concepts/okta-organizations/index.md [hunk 2]
```diff
@@ -10,26 +10,23 @@ Within every org, there are users and applications. These are the only mandatory
...
 
 ## Cells
-Each Okta org exists in a specific segment (or "cell") of Okta's infrastructure. A cell is a conceptual grouping of Okta's public-facing services and UI for a subset of orgs. Cells are completely independent of each other, so failure in one cell doesn't affect a failure in another cell. You can locate which cell your org belongs in by looking at the footer of any page of your Okta administrator console.
+Each Okta org exists in a specific segment (or "cell") of Okta's infrastructure. A cell is a conceptual grouping of Okta's public-facing services and UI for a subset of orgs. Cells are completely independent of each other, so failure in one cell doesn't affect a failure in another cell. You can locate which cell that your org belongs in by looking at the footer of any page of your Okta Administrator Console.
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/docs/concepts/okta-organizations/index.md]: You can locate **the** cell that your org belongs **to** by....

File: packages/@okta/vuepress-site/docs/concepts/okta-organizations/index.md [hunk 3]
```diff
@@ -52,7 +49,7 @@ A Generally Available (GA) feature is new or enhanced functionality that is enab
 
 Early Access (EA) features are opt-in features that you can try out in your org by asking Okta Support to enable them. Additionally, the **Features** page in the Okta Admin Console (**Settings** > **Features**) allows Super Admins to enable and disable some EA features themselves (assuming your org is eligible for the feature). 
 
-You can disable the EA features that you've enabled by deselecting them on the **Features** page. EA features that you disable are re-enabled by Okta automatically when the feature becomes GA.
+You can disable the EA features that you've enabled by clearing the associated boxes on the **Features** page. EA features that you disable are re-enabled by Okta automatically when the feature becomes GA.
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/docs/concepts/okta-organizations/index.md]: ...you've enabled by clearing the associated *check boxes* on the **Features** page.

[According to the MS style guide - "Use check box, not box or checkbox, to refer to a check box in UI."]

**Comment by susanharper-okta** [comment]
> A couple of minor issues

Thanks, Ian. I appreciate it! I'm not doing comprehensive changes to the docs yet on the style issues...just locating them and making note in the style wiki for addressing later. But, totally appreciate you calling out some of them!

**Pull request merged**